### PR TITLE
feat: Added support for redrive_allow_policy

### PR DESCRIPTION
**Pull request by akkigpt**
Need attribute function to add redrive_allow_policy

### CODE REVIEW

**Comment by antonbabenko** [comment]
Please fix the docs by running `pre-commit run -a` and committing the changes. Also, if you can, please update the code in the examples folder to have redrive_allow_policy for real.

**Comment by akkigpt** [comment]
Added example and completed pre-commit

File: examples/complete/main.tf [hunk 1]
```diff
@@ -28,6 +28,22 @@ module "users_encrypted" {
...
 
+
+module "sqs_dlq_allow_redrive_policy" {
+  source = "../../"
+
+  name_prefix = "sqs-dlq-allow-redrive-policy-example"
+
+  redrive_allow_policy = jsonencode({
+    redrivePermission = "byQueue",
+    sourceQueueArns   = ["${module.users_encrypted.sqs_queue_arn}"]
```
Comment (antonbabenko) [inline examples/complete/main.tf:39]:
```suggestion
    sourceQueueArns   = [module.users_encrypted.sqs_queue_arn]
```

**Comment by antonbabenko** [comment]
This PR is included in version 3.3.0 :tada:

**Pull request merged**